examples/brushing/app.js

```diff
@@ -106,6 +115,7 @@ class Root extends Component {
             brushRadius={100000}
             opacity={0.7}
             strokeWidth={2}
+            enableBrushing={true}
```

Comment:
You don't need to specify the value when you want to pass true, but that's a neatpick